WebRoot/css/zm.css

```diff
@@ -1667,7 +1667,31 @@ div.CompDragArea * {
 
 .ZmGrantRightsDialog .addrBubbleContainer,
 .ZmPreferencesPage .addrBubbleContainer {
-    width: 290px;
+	@GrantRightsDialogInputContainer@
+}
+
+.ZmGrantRightsDialog .WindowOuterContainer {
+	@GrantRightsDialogOuterContainer@
+}
+
+.ZmGrantRightsDialog #ZmGrantRightsDialog_name_row .Label {
+	@GrantRightsDialogEmailLabel@
+}
+
+.ZmGrantRightsDialog INPUT[type='text'] {
+	@GrantRightsDialogInput@
+}
+
+.ZmGrantRightsDialog .Label:empty {
+	display:none;
+}
+
+.ZmGrantRightsDialog .SendAs, .ZmGrantRightsDialog .SendObo {
+	padding-top:@SkinWrapperPadding@;
```

Comment:
I donot think using line height will make difference, here.
I prefer using padding as it gives  more control across browsers, as line height can vary on which font you are using, along with what fonts are installed/not installed on the clients browser.